pkgs/development/tools/misc/texinfo/4.13a.nix

```diff
@@ -12,12 +19,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ ncurses ];
   nativeBuildInputs = [ xz ];
 
-  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.cc.isClang [
-    "-Wno-error=implicit-function-declaration"
-  ]);
+  env.NIX_CFLAGS_COMPILE = toString (
+    lib.optionals stdenv.cc.isClang [ "-Wno-error=implicit-function-declaration" ]
+  );
 
   # Disabled because we don't have zdiff in the stdenv bootstrap.
   #doCheck = true;
 
-  meta = texinfo.meta // { branch = version; };
+  meta = texinfo.meta // {
+    branch = version;
+  };
```

Comment:
Also seems pointless to me.

Reply:
[See my comment above](https://github.com/NixOS/nixpkgs/pull/289690#discussion_r1623569633) about nixfmt

Reply:
See the link to "Code conventions" above.